lib/processors/lib/skepticwords.txt

```diff
@@ -0,0 +1,17 @@
+snowden
```

Comment:
Yeah, I agree. Should we take these text files and pass them around the CoLab Slack @mynameisfiber?